test/provider/model/ClientRegistry.unittest.ts

```diff
@@ -84,110 +75,100 @@ describe('When querying windows', () => {
     });
 
     test('Apps start not action-ready', () => {
-        expect(clientRegistry.isAppActionReady(mockUuid1)).toBe(false);
+        expect(clientRegistry.isAppActionReady(fakeUUid1)).toBe(false);
     });
 
     test('When a window adds an action listener, its app, and only its app, becomes action-ready', () => {
-        clientRegistry.onAddEventListener('notification-action', mockApp1Window1);
+        clientRegistry.onAddEventListener('notification-action', fakeApp1Window1);
 
-        expect(clientRegistry.isAppActionReady(mockUuid1)).toBe(true);
-        expect(clientRegistry.isAppActionReady(mockUuid2)).toBe(false);
+        expect(clientRegistry.isAppActionReady(fakeUUid1)).toBe(true);
+        expect(clientRegistry.isAppActionReady(fakeUuid2)).toBe(false);
     });
 
     test('When a window removes its action listener, and it is an app\'s only window, the app becomes not action-ready', () => {
-        clientRegistry.onAddEventListener('notification-action', mockApp1Window1);
-        clientRegistry.onRemoveEventListener('notification-action', mockApp1Window1);
+        clientRegistry.onAddEventListener('notification-action', fakeApp1Window1);
+        clientRegistry.onRemoveEventListener('notification-action', fakeApp1Window1);
 
-        expect(clientRegistry.isAppActionReady(mockUuid1)).toBe(false);
+        expect(clientRegistry.isAppActionReady(fakeUUid1)).toBe(false);
     });
 
     test('When a window with an action listener disconnects, the app becomes not action-ready', () => {
-        clientRegistry.onAddEventListener('notification-action', mockApp1Window1);
+        clientRegistry.onAddEventListener('notification-action', fakeApp1Window1);
 
-        mockApiHandler.onDisconnection.emit(mockApp1Window1);
+        mockApiHandler.onDisconnection.emit(fakeApp1Window1);
 
-        expect(clientRegistry.isAppActionReady(mockUuid1)).toBe(false);
+        expect(clientRegistry.isAppActionReady(fakeUUid1)).toBe(false);
     });
 
     test('When two windows of the same app add an action listener, and then one removes its action listener, the app stays action-ready', () => {
-        clientRegistry.onAddEventListener('notification-action', mockApp1Window1);
-        clientRegistry.onAddEventListener('notification-action', mockApp1Window2);
+        clientRegistry.onAddEventListener('notification-action', fakeApp1Window1);
+        clientRegistry.onAddEventListener('notification-action', fakeApp1Window2);
 
-        clientRegistry.onRemoveEventListener('notification-action', mockApp1Window1);
+        clientRegistry.onRemoveEventListener('notification-action', fakeApp1Window1);
 
-        expect(clientRegistry.isAppActionReady(mockUuid1)).toBe(true);
+        expect(clientRegistry.isAppActionReady(fakeUUid1)).toBe(true);
     });
 
     test('When two windows of the same app add an action listener, and then both removes their action listener, the app becomes not action-ready', () => {
```

Comment:
Not your change and not really important at all, but this should be "remove" not "removes"

Reply:
✅